ml-agents-envs/mlagents_envs/side_channel/utils.py

```diff
@@ -0,0 +1,68 @@
+import uuid
+import struct
+from typing import Dict, Optional, List
+from mlagents_envs.side_channel import SideChannel, IncomingMessage
+from mlagents_envs.exception import UnityEnvironmentException
+from mlagents_envs.logging_util import get_logger
+
+
+def parse_side_channel_message(
```

Comment:
consume_side_channel_messages? 
Add a docstring indicating that this fires the on_message_received() callbacks.

Reply:
On C# in the SideChannelManager: ProcessSideChannelData and GetSideChannelMessage
I can mirror those?